src/observers/stores/datasets.py

```diff
@@ -55,7 +55,7 @@ def _cleanup(self):
             shutil.rmtree(self._temp_dir)
 
     def _init_table(self, record: "Record"):
-        repo_name = self.repo_name or record.table_name
+        repo_name = self.repo_name or "my-dataset"
```

Comment:
why do you want my-dataset to be the default?

Reply:
`openai_records` isn't always representative of the dataset. Any better ideas?

Reply:
Yeah I guess openai is unique since some of the time records may not be openai but could be llama-cpp, huggingface, vllm etc.

I think the other stores Argilla etc could have those specific names

Reply:
similar to how wandb does their training runs (_left side_)

![image](https://github.com/user-attachments/assets/df5ca0d4-f5ee-4f84-abc0-8149afeb1dbd)